### PR TITLE
Add support for Rathpith Globe and legacy Femur of the Saints

### PR DESCRIPTION
**Pull request by Wires77**

### CODE REVIEW

File: src/Modules/ModParser.lua [hunk 1]
```diff
@@ -981,6 +982,7 @@ local modTagList = {
 	["per (%d+) evasion rating, up to (%d+)%%"] = function(num, _, limit) return { tag = { type = "PerStat", stat = "Evasion", div = num, limit = tonumber(limit), limitTotal = true } } end,
 	["per (%d+) maximum energy shield"] = function(num) return { tag = { type = "PerStat", stat = "EnergyShield", div = num } } end,
 	["per (%d+) maximum life"] = function(num) return { tag = { type = "PerStat", stat = "Life", div = num } } end,
+	["per (%d+) player maximum life"] = function(num) return { tag = { type = "PerStat", actor = "Player", stat = "Life", div = num, actor = "parent" } } end,
```
Comment (PJacek) [inline src/Modules/ModParser.lua:985]: `actor` is doubled and there is no `Player` actor

Comment (Wires77) [reply src/Modules/ModParser.lua:985]: Thanks, forgot to remove that bit after I was done

File: src/Modules/ModParser.lua [hunk 2]
```diff
@@ -1010,9 +1012,12 @@ local modTagList = {
 	["per summoned golem"] = { tag = { type = "PerStat", stat = "ActiveGolemLimit" } },
 	["per summoned sentinel of purity"] = { tag = { type = "PerStat", stat = "ActiveSentinelOfPurityLimit" } },
 	["per summoned skeleton"] = { tag = { type = "PerStat", stat = "ActiveSkeletonLimit" } },
+	["per skeleton you own"] = { tag = { type = "PerStat", stat = "ActiveSkeletonLimit", actor = "parent" } },
 	["per summoned raging spirit"] = { tag = { type = "PerStat", stat = "ActiveRagingSpiritLimit" } },
 	["for each raised zombie"] = { tag = { type = "PerStat", stat = "ActiveZombieLimit" } },
+	["per zombie you own"] = { tag = { type = "PerStat", stat = "ActiveZombieLimit", actor = "parent" } },
 	["per raised spectre"] = { tag = { type = "PerStat", stat = "ActiveSpectreLimit" } },
+    ["per spectre you own"] = { tag = { type = "PerStat", stat = "ActiveSpectreLimit", actor = "parent" } },
```
Comment (PJacek) [inline src/Modules/ModParser.lua:1020]: Indented with spaces

**Review by ppoelzl — APPROVED** [review]
Nice, we can refactor this later to make it work with all tag types, should the need arise.

**Pull request merged**